cmd/kubelet/app/server.go

```diff
@@ -606,7 +606,7 @@ func run(ctx context.Context, s *options.KubeletServer, kubeDeps *kubelet.Depend
 	}
 
 	var cgroupRoots []string
-	nodeAllocatableRoot := cm.NodeAllocatableRoot(s.CgroupRoot, s.CgroupsPerQOS, s.CgroupDriver)
+	nodeAllocatableRoot := cm.NodeAllocatableRoot(s.CgroupRoot, s.CgroupsPerQOS, s.CgroupDriver == "systemd")
```

Comment:
Can we keep the old cgroup driver signature here as well, and then do the refactor in another one as suggested by @ehashman (or did you just suggest for a single instance)?

Reply:
Done.